frontend/src/_components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -0,0 +1,297 @@
+import React, { useState, useRef, useEffect, useMemo } from 'react';
+import DatePickerComponent from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+import cx from 'classnames';
+import { ButtonSolid } from '@/_ui/AppButton/AppButton';
+import SolidIcon from '@/_ui/Icon/SolidIcons';
+import './styles.scss';
+import { convertToDateType, formatDate, getLocalTimeZone } from '@/_helpers/utils';
+
+export const DateTimePicker = ({
```

Comment:
This component is specific for TJDB. Can we create this file within TJDB dashboard directory instead?